<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecCinescenie</class>
 <widget class="QWidget" name="ExecCinescenie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="FrameCinescenie">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="LabelCinescenie">
        <property name="text">
         <string>Cinéscénie :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelCinescenieName">
        <property name="text">
         <string>CinescenieName</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="HSpacerCinescenie">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>595</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonClose">
        <property name="text">
         <string>Fermer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="FrameDefineTimecode">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="LabelDefineTimecode">
        <property name="text">
         <string>Définir le time code :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="EditTimecode">
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonStart">
        <property name="text">
         <string>Démarrer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="HSpacerTimecode">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>564</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="FrameEvent">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="LabelNextEvent">
        <property name="text">
         <string>Prochain évènement :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelNextEventTimecode">
        <property name="text">
         <string>NextEventTimecode</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="HSpacerEvent">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>460</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="LabelTimecode">
        <property name="text">
         <string>Time code :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelCurrentTimecode">
        <property name="text">
         <string>CurrentTimeCode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="TableEvents">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Time code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temps restant</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Annonce</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutVocalAnnounce">
     <item>
      <spacer name="HSpacerVocalAnnounce">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="CheckboxVocalAnnounce">
       <property name="text">
        <string>Activer les annonces vocales</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
